server/pbenchinacan/load_keycloak.sh

```diff
@@ -67,9 +67,9 @@ fi
 
 # Create a client scope with custom mapper that a functional test
 # user can include in the OIDC token request over a REST call.
-# This will instruct Keycloak to include the CLIENT_ID (pbench-dashboard)
-# when someone request a token Over a rest API using a CLIENT_ID.
-# Having CLIENT_ID in the aud claim of the token is essential for the token
+# This will instruct Keycloak to include the <client_id> (pbench-dashboard)
+# when someone request a token Over a rest API using a <client_id>.
```

Comment:
"when someone requests a token from Keycloak"??? The capitalized "Over" and lowercase "rest" is distracting here, and I think the whole thing could probably be less "busy".